packages/components/progress-bar/src/progress-bar.ts

```diff
@@ -66,6 +67,23 @@ export class ProgressBar extends ScopedElementsMixin(LitElement) {
     }
   }
 
+  override updated(changes: PropertyValues<this>): void {
+    super.updated(changes);
+
+    if (changes.has('value')) {
```

Comment:
`variant` change should be also announced, right?

Reply:
But not with every value, that will get annoying pretty fast...